CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 Yii Framework 2 composer extension Change Log
 =============================================
 
-2.0.9 under development
+2.1.0 under development
```

Comment:
I think this should be 2.0.9, there is nothing breaking here and we can migrate to composer 2.0 without the need to adjust the requirement in yiisoft/yii2

```suggestion
2.0.9 under development
```

Reply:
Fine with me! I made that change because of your comment at https://github.com/yiisoft/yii2-composer/issues/27#issuecomment-611028574.

I can’t change it back at this point without creating another PR though, so I’ll leave it to you.